lib/src/message.dart

```diff
@@ -134,17 +143,48 @@ abstract class Message {
 ///
```

Comment:
Done in `new Request()` and `new Response()` (the `Message` class isn't actually public).